packages/amplify-graphql-transformer-interfaces/src/transformer-context/transformer-datasource-provider.ts

```diff
@@ -6,7 +6,7 @@ import { InterfaceTypeDefinitionNode, ObjectTypeDefinitionNode } from 'graphql';
 
 export enum AppSyncDataSourceType {
   AMAZON_DYNAMODB = 'AMAZON_DYNAMODB',
-  AMAZON_ELASTICSEARCH = 'AMAZON_ELASTICSEARCH',
+  AMAZON_OPENSEARCH = 'AMAZON_OPENSEARCH',
```

Comment:
I'd be surprised if this works. I'm not sure if we can change this yet, as we'd need `AMAZON_OPENSEARCH` to be supported in AppSync and CFN.